packages/ui/hooks/index.ts

```diff
@@ -74,6 +74,27 @@ export function useIsMounted() {
    return isMounted;
 }
 
+const isClient = typeof window !== 'undefined';
+let hydrationComplete = false;
```

Comment:
What is the purpose of the `hydrationComplete` variable?

Reply:
I think the below comment actually addresses this.